runelite-client/src/main/java/net/runelite/client/plugins/statusorbs/StatusOrbsPlugin.java

```diff
@@ -243,4 +443,28 @@ int getEstimatedRecoverTimeRemaining()
 		final double secondsLeft = (100 - client.getEnergy()) / recoverRate;
 		return (int) secondsLeft;
 	}
+
+	/**
+	 * Migrates configs from runenergy and regenmeter to this plugin and deletes the old config values.
+	 */
+	private void migrateConfigs()
```

Comment:
This should be annotated `@Deprecated`, and have a note regarding removing it after some time. See [`FarmingTracker.migrateConfiguration()` from the time tracking PR](https://github.com/runelite/runelite/pull/4510/files#diff-98debac5d25615bb4494938070777f37R237) or [`GroundMarkerPlugin.setColorConfiguration()` from my colored ground markers PR](https://github.com/runelite/runelite/pull/5890/files#diff-c65443644d54b889eb2a4afdcaa07550R255).